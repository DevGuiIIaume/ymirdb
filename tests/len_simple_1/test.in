set z 1
len z
len a
bye
