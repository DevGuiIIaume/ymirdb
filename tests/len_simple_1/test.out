> ok

> 1

> no such key

> bye
